app/models/main.py

```diff
@@ -1832,6 +1848,302 @@ def serialize(self):
         }
 
 
+class ProcessOutcome(db.Model):
+    __tablename__ = "process_outcomes"
+
+    RESULT_CHOICES = (
+        "awarded",
+        "cancelled",
+        "none-suitable",
+    )
+
+    id = db.Column(db.Integer, primary_key=True)
+    external_id = db.Column(db.BigInteger, nullable=False, default=random_positive_external_id, unique=True)
+    completed_at = db.Column(db.DateTime, nullable=True)
+
+    start_date = db.Column(db.Date, nullable=True)
+    end_date = db.Column(db.Date, nullable=True)
+    awarding_organisation_name = db.Column(db.String, nullable=True)
+    # should be enough scale to represent up to £1b - 1p
+    award_value = db.Column(db.Numeric(11, 2), nullable=True)
```

Comment:
This will have created a float won't it - don't you need `asdecimal=True`?

Reply:
Um.... let me check that - I didn't *think* so when I was playing with it...

Reply:
Yeah, `asdecimal=True` is the default: http://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.Numeric